src/crimson/CMakeLists.txt

```diff
@@ -170,6 +170,8 @@ add_library(crimson STATIC
 target_compile_options(crimson PUBLIC
   "-ftemplate-backtrace-limit=0")
 target_link_libraries(crimson
+  PRIVATE
+    pthread
```

Comment:
i am trying to fix this on the seastar side. see https://groups.google.com/d/msg/seastar-dev/YpZcue4-I2U/8xBDbmO7BAAJ